docs/docs/develop/setup.md

```diff
@@ -54,10 +60,17 @@ tags:
 | `--version`       | Show version number                        | boolean                                                         |
 | `--directory`     | Output directory                           | string [default: "/users/bennyp/developer/patternfly-elements"] |
 | `--silent`        | Do not log anything to stdout              | boolean [default: false]                                        |
-| `-n`, `--tagName` | Custom element tag name. e.g. `pf-button` | string                                                          |
+| `-n`, `--tagName` | Custom element tag name. e.g. `pf-button`  | string                                                          |
 | `-s`, `--scope`   | NPM package scope. e.g. `@patternfly`      | string                                                          |
 | `--overwrite`     | Overwrite files without prompting          | boolean [default: false]                                        |
```

Comment:
There's another option in the tool for `--css` which says it can set up `postcss` files optionally but I can't seem to get it to work.  @bennypowers Is this deprectated / can it be removed in a separate PR?

Reply:
That would be a major version. Better to fix it